SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier2/JetBoots.cs

```diff
@@ -42,8 +42,9 @@ public sealed class JetBoots : SS2Item
         private static GameObject _tracerPrefab;
         private static GameObject _muzzleFlashPrefab;
         private static GameObject _effectPrefab;
-        private static BuffDef _buffCooldown;
-    
+        private static BuffDef _buffCooldown; // SS2Assets.LoadAsset<BuffDef>("BuffJetBootsCooldown", SS2Bundle.Items);
+        private static BuffDef _buffReady; //SS2Assets.LoadAsset<BuffDef>("BuffJetBootsReady", SS2Bundle.Items);
+
         public override void Initialize()
         {            // this will interfere with other bonus jump items but they can be unified in a similar way to this
             IL.RoR2.CharacterBody.RecalculateStats += RecalculateStatsHook; // recalculatestatsapi doesnt have maxjumpcount
```

Comment:
Look into the possibility of moving this to actual recalculate stats API as a new feature.